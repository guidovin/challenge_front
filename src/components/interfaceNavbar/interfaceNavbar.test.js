import React from 'react';
import { shallow } from 'enzyme';

import InterfaceNavbar from './index.js';

describe('InterfaceNavbar', () => {
  it('should render correctly in "debug" mode', () => {

    const component = shallow(<InterfaceNavbar />);
    expect(component.find('.prompt')).toBeTruthy();
    expect(component.find('.prompt').props('placeholder').placeholder).toEqual('Todo name...');
    expect(component.find('.viewTogle')).toBeTruthy();

    expect(component).toMatchSnapshot();
  });
});
