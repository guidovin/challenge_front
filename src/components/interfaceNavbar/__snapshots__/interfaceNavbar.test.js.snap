// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InterfaceNavbar should render correctly in "debug" mode 1`] = `
<Menu>
  <MenuItem
    active={false}
    className="viewTogle"
    name="showCompleted"
    onClick={[Function]}
    position="left"
  >
    Showing active
  </MenuItem>
  <MenuMenu
    className="todoInput"
    position="right"
  >
    <div
      className="ui right aligned category search item todoInput"
    >
      <div
        className="ui transparent icon input todoInput"
      >
        <input
          className="prompt"
          id="inputText"
          name="todoText"
          onChange={[Function]}
          onKeyUp={[Function]}
          placeholder="Todo name..."
          type="text"
        />
      </div>
    </div>
    <i
      className="right arrow link icon submit"
      onClick={[Function]}
    />
  </MenuMenu>
</Menu>
`;
